.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: alfresco Extension Inspector Build Workflow
+
+on:
+  push:
+    branches:
+      - master
+      - 'fix/**'
+      - 'support/**'
+      - 'feature/**'
+  pull_request:
+    branches:
+      - master
+
+env:
+  SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    if: >
+      !contains(github.event.head_commit.message, '[skip tests]')
+    strategy:
+      matrix:
+        script: [ source_clear, unit_tests ]
+    steps:
+      # Typical Java workflow steps
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: adopt
+          java-version: 17
+
+      - name: Setup maven
+        shell: bash
+        run: |
+          [ -d ${HOME}/.m2/ ] || mkdir -p ${HOME}/.m2/
+          cp -v _ci/settings.xml ${HOME}/.m2/ || cp -v .travis.settings.xml ${HOME}/.m2/settings.xml
+
+      - name: Cache the Maven packages to speed up build
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+
+      - name: Run ${{ matrix.script }}
+        run:  bash _ci/${{ matrix.script }}.sh
+
+  Release:
+    name: "Release"
+    runs-on: ubuntu-latest
+    needs: tests
+    if: >
+      contains(github.event.head_commit.message, '[release]') && 
+      github.ref_name == 'master' && 
+      github.event.repository.fork==false
+    steps:
+      # Typical Java workflow steps
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.8.10
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: "Configure AWS credentials"
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_KEY }}
+          aws-region: eu-west-1
+      - name: "prepare_staging_deploy"
+        run: bash _ci/prepare_staging_deploy.sh
+      - name: "Deploy to S3 Staging Bucket"
+        run: aws s3 cp --recursive ./deploy_dir s3://alfresco-artefacts-staging
```

Comment:
be consistent with spacing between steps, I prefer a blank line after each step (like it is on the previous job)